<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>827</width>
    <height>593</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>827</width>
    <height>593</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>100</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="Students">
        <attribute name="title">
         <string>Абитуриенты</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>800</width>
           <height>531</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="StudLayout"/>
        </widget>
       </widget>
       <widget class="QWidget" name="__FacGroup">
        <attribute name="title">
         <string>Факультеты/Специальности/Формы обучения</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>800</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="FacLayout"/>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>800</width>
           <height>150</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="SpecLayout"/>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>350</y>
           <width>800</width>
           <height>180</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="FormLayout"/>
        </widget>
       </widget>
       <widget class="QWidget" name="__Subjects">
        <attribute name="title">
         <string>Договора</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>800</width>
           <height>501</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="ContrLayout"/>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
